content/developer/Theme/Icons/SuitePIconsFont/_index.en.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: "SuiteP Icons Font"
+weight: 20
+pre: "<b> </b>"
+---
+
+=== Table of Contents
+{{% children depth="4" showhidden="false" %}}
```

Comment:
The same as commented for the restructuring of the `SuiteP Icons` page. Please pull in the 3 sub-pages as headers inside this one.